clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -286,11 +289,17 @@ pt2ir_t::convert(DR_PARAM_IN const uint8_t *pt_data, DR_PARAM_IN size_t pt_data_
 
     /* This flag indicates whether manual synchronization is required. */
     bool manual_sync = true;
-
     /* PT raw data consists of many packets. And PT trace data is surrounded by Packet
      * Stream Boundary. So, in the outermost loop, this function first finds the PSB. Then
      * it decodes the trace data.
      */
+    uint64_t decoded_instr_count = 0;
+    uint64_t recoverable_error_count = 0;
+    /* XXX: This is currently set based on empirical observations. We use this heuristic
+     * to detect recoverable errors: any non-consecutive error of type pte_bad_query, and
+     * errors up to this limit are attempted to recover from by retrying.
```

Comment:
This sounds like we allow any type of error other than pte_bad_query up to this limit: but the code *only* allows pte_bad_query.  Please re-word: the code allows up to kErrorLimit instances of non-consecutive pte_bad_query.  If that's the intent: please make this comment match that.

Reply:
Done.